app-text/ghostscript_gpl/ghostscript_gpl-9.25.recipe

```diff
@@ -0,0 +1,131 @@
+languages. Its main purposes are the rasterization or rendering of such page \
+description language files, for the display or printing of document pages, and \
+the conversion between PostScript and PDF files."
+HOMEPAGE="https://ghostscript.com/"
+COPYRIGHT="2001-2018 Artifex Software, Inc."
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs925/ghostscript-$portVersion.tar.gz"
+CHECKSUM_SHA256="baafa64740b090bff50b220a6df3be95c46069b7e30f4b4effed28316e5b2389"
+SOURCE_DIR="ghostscript-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	ghostscript_gpl$secondaryArchSuffix = $portVersion
+	cmd:dvipdf$commandSuffix
+	cmd:eps2eps$commandSuffix
+	cmd:font2c$commandSuffix
+	cmd:gs$commandSuffix
+	cmd:gsbj$commandSuffix
+	cmd:gsdj$commandSuffix
+	cmd:gsdj500$commandSuffix
+	cmd:gslj$commandSuffix
+	cmd:gslp$commandSuffix
+	cmd:gsnd$commandSuffix
+	cmd:lprsetup.sh$commandSuffix
+	cmd:pfbtopfa$commandSuffix
+	cmd:pdf2dsc$commandSuffix
+	cmd:pdf2ps$commandSuffix
+	cmd:pf2afm$commandSuffix
+	cmd:pphs$commandSuffix
+	cmd:printafm$commandSuffix
+	cmd:ps2ascii$commandSuffix
+	cmd:ps2epsi$commandSuffix
+	cmd:ps2pdf$commandSuffix
+	cmd:ps2pdfwr$commandSuffix
+	cmd:ps2pdf12$commandSuffix
+	cmd:ps2pdf13$commandSuffix
+	cmd:ps2pdf14$commandSuffix
+	cmd:ps2ps$commandSuffix
+	cmd:ps2ps2$commandSuffix
+	cmd:unix_lpr.sh$commandSuffix
+	cmd:wftopfa$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	poppler_data
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libidn$secondaryArchSuffix
+	lib:libjbig2dec$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblcms2$secondaryArchSuffix
+	lib:libpaper$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libidn$secondaryArchSuffix
+	devel:libjbig2dec$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblcms2$secondaryArchSuffix
+	devel:libpaper$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+defineDebugInfoPackage ghostscript_gpl$secondaryArchSuffix \
+	"$commandBinDir"/gs
+
+BUILD()
+{
+	# Set up fonts paths
+	local FONTPATH
+	for path in \
+		$fontsDir/otfonts \
+		$fontsDir/psfonts \
+		$fontsDir/ttfonts \
+		$fontsDir/X11
```

Comment:
```diff
-		$fontsDir/otfonts \
-		$fontsDir/psfonts \
-		$fontsDir/ttfonts \
-		$fontsDir/X11
+		$(findpath B_FIND_PATH_FONTS_DIRECTORY otfonts) \
+		$(findpath B_FIND_PATH_FONTS_DIRECTORY psfonts) \
+		$(findpath B_FIND_PATH_FONTS_DIRECTORY ttfonts) \
+		$(findpath B_FIND_PATH_FONTS_DIRECTORY X11)
```

If `fontDir` is used, then the program can only read it's own fonts. This is still considered a hack because the paths are hard-coded into the executable, but we can open an issue to fix that later.

Reply:
Fixed.